README.md

```diff
@@ -41,45 +41,174 @@ To communicate with CoolWallet device, you need to specify a bluetooth transport
+| [`@coolwallet/dot`](/packages/cws-dot) | ![version](https://img.shields.io/npm/v/@coolwallet/dot) | Polkadot/Kusama |
+| [`@coolwallet/eth`](/packages/cws-eth) | ![version](https://img.shields.io/npm/v/@coolwallet/eth) | Ethereum (Ether, ERC20, Smart Contract, EIP-1559 etc.)|
+| [`@coolwallet/icx`](/packages/cws-icx) | ![version](https://img.shields.io/npm/v/@coolwallet/icx) | Icon |
+| [`@coolwallet/ltc`](/packages/cws-ltc) | ![version](https://img.shields.io/npm/v/@coolwallet/ltc) | LetCoin |
+| [`@coolwallet/trx`](/packages/cws-trx) | ![version](https://img.shields.io/npm/v/@coolwallet/trx) | Tron |
+| [`@coolwallet/xlm`](/packages/cws-xlm) | ![version](https://img.shields.io/npm/v/@coolwallet/xlm) | Stellar/Kinesis |
+| [`@coolwallet/xrp`](/packages/cws-xrp) | ![version](https://img.shields.io/npm/v/@coolwallet/xrp) | Ripple |
+| [`@coolwallet/zen`](/packages/cws-zen) | ![version](https://img.shields.io/npm/v/@coolwallet/zen) | Zen Cash |
 
-Other supported coins: BTC, BCH, ZEN. Open an issue if you want the sdk of any one of them to come out first.
 
-### Other packages
+## Examples: Build ETH in web app
+### Create connect
 
-| Package                                 | Version                                                   | Description              |
-| --------------------------------------- | --------------------------------------------------------- | ------------------------ |
-| [`@coolwallet/web3-subprovider`](/packages/web3-subprovider) | ![version](https://img.shields.io/npm/v/@coolwallet/web3-subprovider) | Web3 subprovoder that can be use with [web3-provider-engine](https://github.com/MetaMask/web3-provider-engine) |
 
+```
+npm install @coolwallet/core
+npm install @coolwallet/transport-web-ble
+```
+
+```javascript
+import WebBleTransport from "@coolwallet/transport-web-ble";
+import * as core from "@coolwallet/core";
+```
+
+Create a connection to obtain the Card Name and SE Public Key.
+
+```javascript
+
+connect = async () => {
+WebBleTransport.listen(async (error, device) => {
+    const cardName = device.name;
+    const transport = await WebBleTransport.connect(device);
+    const SEPublicKey = await core.config.getSEPublicKey(transport)
+    this.setState({ transport, cardName, SEPublicKey });
+    localStorage.setItem('cardName', cardName)
+    localStorage.setItem('SEPublicKey', SEPublicKey)
+  });
+};
+
+disconnect = () => {
+  WebBleTransport.disconnect(this.state.transport.device.id);
+  this.setState({ transport: undefined, cardName: "" });
+};
+
+```
+
+- transport: The object use to communicate with CoolWallet
+- SEPublicKey: The key use to check se.
+
+Obtain app key pairs.
+
+```javascript
+const keyPair = crypto.key.generateKeyPair()
+localStorage.setItem('appPublicKey', keyPair.publicKey)
+localStorage.setItem('appPrivateKey', keyPair.privateKey)
+```
+
+- keyPair: The keys use to check your app.
```

Comment:
shall we move this part to next section?

Reply:
I will move this code to `Register application with CoolWallet Pro`